config/config.properties

```diff
@@ -91,6 +91,22 @@ pravegaservice.dataLogImplementation=BOOKKEEPER
 # Default value: HDFS
 # pravegaservice.storageImplementation=HDFS
 
+# Storage NO-OP Mode: segment payload write is no-oped; payload reading is not supported
+# This mode is used to avoid tier 2 interference in testing while keep the system functioning as usual.
+#
+# NOTE: pravegaservice.storageImplementation is still used to store metadata.
+# WARNING: Do NOT set NO-OP Mode to true in production environment!!! Data loss will happen!!!
+#
+# Valid values: true, false
+# Default value: false
+#storageextra.storageNoOpMode=false
```

Comment:
Because the No-Op implementation cannot replace the Tier 2 adapter. It needs a Tier 2 adapter as pass-through so that `_system` segments can function normally (i.e., they should not be affected). If we also ignore all operations on `_system` segments then the Controller may not be able to retrieve its metadata and the cluster would become unusable after the first Segment Container failover & recovery. Passing-through the `_system` segments has no impact on measuring performance (they hold a combined total of a few MBs at most) but are required for the good functioning of Pravega.

Reply:
Put this property into a separate config component also improves the isolation - in case Storage is not in No-Op mode, which is in most cases, then the entire NoOpStorage, including relevant config properties, are completely off the scene.

Reply:
The prefix is the name space for the properties. Similarly for FileSystemStorage, S3Storage, HDFSStorage, their specific properties are all put under different name space. Otherwise the pravegaservice namespace is going to be very crowded; but more importantly, under the same name space, we may have to still differentiate properties by names such as "latencyForS3", "latencyForHDFS" for those common properties across storage.